rockcraft/models/project.py

```diff
@@ -441,7 +441,8 @@ def generate_metadata(
             "org.opencontainers.image.ref.name": self.name,
             "org.opencontainers.image.created": generation_time,
             "org.opencontainers.image.base.digest": base_digest.hex(),
-            "org.opencontainers.image.description": f"{self.summary}\n\n{self.description}",
+            "org.opencontainers.image.description": re.sub(r"\n{2,}", " ", self.summary)
```

Comment:
shouldn't we replace the sequence of newlines with a single newline? It would preserve the original structure a bit better

Reply:
sure. I was just doing a `replace` originally, so the " " was to cope with more that 2 consecutive `\n`. but `{2,}` takes care of that. changed